src/store/reducers.tsx

```diff
@@ -1,32 +1,25 @@
 const CARD_TEXTS = require('./values.json')
 // Update once DefinitelyTyped includes uuid 3.0.1
 const uuid = require('uuid');
-import { combineReducers, createStore } from 'redux';
+import { createStore } from 'redux';
 
 
 /*
-Actions:
-
-MARK_CARD
-ADVANCE_STAGE
-
+  Actions:
+    TOGGLE_CARD
+    ADVANCE_STAGE
+    RETREAT_STAGE
 */
 
-const card = (state: any = {}, action: any) => {
+const card = (state: any = {}, action: any, stage: number) => {
```

Comment:
Because of Redux patterns, `stage` should be a property on `action`, instead of as a separate parameter on the reducer function.